dvc/command/run.py

```diff
@@ -8,6 +8,13 @@ def run(self):
             if self.args.yes:
                 self.project.prompt.default = True
 
+            if len(self.args.outs) != len(set(self.args.outs)):
+                s = set([x for x in self.args.outs if self.args.outs.count(x) > 1])
+                for i in s:
+                    print("Warning: Output \'" + i +
```

Comment:
I think this is worth raising an exception. Please see `_check_output_duplication` function at https://github.com/iterative/dvc/blob/master/dvc/project.py#L136. And please move this logic to dvc/project.py